core/src/main/java/nu/yona/server/subscriptions/service/BuddyConnectResponseMessageDto.java

```diff
@@ -137,6 +140,7 @@ MessageActionDto handleAction_Process(UserDto actingUser, BuddyConnectResponseMe
 						connectResponseMessageEntity.getRelatedUserAnonymizedId().get(),
 						connectResponseMessageEntity.getSenderNickname());
 			}
+			UserDto actingUserAfterUpdate = userService.getPrivateUser(actingUser.getId());
```

Comment:
Why is this being fetched again? Is that related to my previous comment about the removed comment?

And why is the user fetched here already and only used 12 lines later?